neurons/miners/miner.py

```diff
@@ -292,13 +275,12 @@ def set_weights(self):
 
         bt.logging.info(f"Set weights: {chain_weights}")
     
-    # Make storing metadata configurable
     def should_send_metadata(self):
-        miner_config = MinerConfig().load_and_get_config_values()
+        self.miner_config
```

Comment:
is this required? self.miner_config is updated before running should_send_metadata
its updated while resync

Reply:
It is, we update miner configuration in the presence metagraph